packages/bpk-component-star-rating/src/BpkStarRating.js

```diff
@@ -77,13 +91,19 @@ BpkStarRating.propTypes = {
   large: PropTypes.bool,
   maxRating: PropTypes.number,
   rating: PropTypes.number,
+  rounding: PropTypes.oneOf([
+    ROUNDING_TYPES.down,
+    ROUNDING_TYPES.up,
+    ROUNDING_TYPES.nearest,
+  ]),
```

Comment:
```suggestion
  rounding: PropTypes.oneOf(Object.keys(ROUNDING_TYPES)),
```

Reply:
I changed this to `Object.values`, if I use `Object.keys` it expects `[down, up, nearest]`

Reply:
Ah yeah of course it does 🤦‍♂ 
Ignore me 😆

Reply:
`Object.values` isn't supported by all browsers, so best to roll with what you had before
https://caniuse.com/#search=Object.values

Reply:
Good catch! Will do 👍